uPortal-webapp/src/main/webapp/scripts/portal-analytics.js

```diff
@@ -42,9 +42,16 @@
             return;
         }
 
+        const tab = document.querySelector('li.fl-tabs-active span');
```

Comment:
It looks like ESLint is unhappy https://github.com/uPortal-Project/uPortal/actions/runs/5954783612/job/16152048170?pr=2694#step:6:221

This is likely due to a preset defaulting to an older version of JavaScript
Try setting `env` with` `browser` and `es2023` https://eslint.org/docs/latest/use/configure/language-options

https://github.com/uPortal-Project/uPortal/blob/15de05e2d7bee8a55d20f0560dc23c0e9a801025/package.json#L26C4-L60